service/src/main/kotlin/no/nav/su/se/bakover/service/statistikk/Statistikk.kt

```diff
@@ -64,11 +64,17 @@ sealed class Statistikk {
         val behandlingOpprettetTypeBeskrivelse: String? = null,
         val datoForUttak: String? = null,
         val datoForUtbetaling: String? = null,
-    ) : Statistikk()
+        val avsluttet: Boolean,
+    ) : Statistikk() {
+        enum class BehandlingType(val beskrivelse: String) {
+            SOKNAD("Søknad for SU Uføre"),
```

Comment:
Som nevnt i en annen kommentar, kunne det gitt mening med fler datamodeller her? søknad føles malplassert her.

Reply:
Litt av forvirringen her kommer nok fra at skjemaene bare definerer to "ting", nemlig sak og behandling.. hendelsen "søknad mottatt" føles vel kanskje ikke naturlig å putte inn noen av stedene? I utgangspunktet heller jeg vel egentlig mot at "søknad mottatt" er en hendelse på saken, og dermed heller burde publisert "noe" ved å bruke det skjemaet - på den andre siden føles det ikke som det passer inn der heller hvis man ser over feltene og beskrivelsene i skjemaet.

Reply:
@aryani1

Reply:
Ja, det Jacob sa 😊  Blev å putte den bland behandlingarna eftersom fälten de bruker stämde överens bättre.